src/test/java/com/rocketden/main/socket/RoomSocketTests.java

```diff
@@ -159,4 +161,22 @@ public void socketReceivesMessageOnHostChange() throws Exception {
         assertEquals(expected.getUsers(), actual.getUsers());
     }
 
+    @Test
+    public void socketReceivesMessageOnSettingsChange() throws Exception {
```

Comment:
If the difficulty is just set to its current setting, should we still send an update?

I think it'd be fine to, but I just wanted to bring up the question.

Reply:
Personally I'm okay with either, if no settings change then it'd just send a message and the front-end would ignore it (I think changing room hosts to the same person causes the same thing). I'd be open to implementing a check tho, just lmk

Reply:
No, I don't think there's any need for that, just wanted to bring it up in case. Thanks!